### PR TITLE
KAFKA-13455: Add steps to run Kafka Connect to quickstart

### PR DESCRIPTION
**Pull request by katheris**
Signed-off-by: Katherine Stanley <11195226+katheris@users.noreply.github.com>

### CODE REVIEW

File: docs/quickstart.html [hunk 1]
```diff
@@ -158,15 +158,78 @@ <h4 class="anchor-heading">
         </h4>
 
         <p>
-            You probably have lots of data in existing systems like relational databases or traditional messaging systems,
-            along with many applications that already use these systems.
-            <a href="/documentation/#connect">Kafka Connect</a> allows you to continuously ingest
-            data from external systems into Kafka, and vice versa.  It is thus very easy to integrate existing systems with
-            Kafka. To make this process even easier, there are hundreds of such connectors readily available.
+            Reading data from the console and writing it back to the console is a convenient place to start, but you'll probably want
```
Comment (mimaison) [inline docs/quickstart.html]: Should we keep the existing paragraph and add this after it? I think it helps introduce Connect.

**Review by mimaison — COMMENTED** [review]
Thanks for the PR! I left a suggestion.

File: docs/quickstart.html (hunk 1 above)
Comment (katheris) [reply docs/quickstart.html]: @mimaison thanks for your review. I think the existing paragraph and the new ones cover similar things so I've had a go at combining them. Let me know if you think it reads ok

**Review by mimaison — APPROVED** [review]
I rendered the docs and tested the steps locally.
LGTM, thanks for the PR!

**Comment by mimaison** [comment]
@katheris Let's also update the website, otherwise we'll have to wait for the next release. Can you open a similar PR on https://github.com/apache/kafka-site? Thanks

**Pull request merged**